iocage/cli/start.py

```diff
@@ -34,22 +36,22 @@
 @click.option("--rc", default=False, is_flag=True,
               help="Will start all jails with boot=on, in the specified"
                    " order with smaller value for priority starting first.")
+@click.option("--log-level", "-d", default=None)
 @click.argument("jails", nargs=-1)
-def cli(rc, jails):
+def cli(rc, jails, log_level):
     """
-    Looks for the jail supplied and passes the uuid, path and configuration
-    location to start_jail.
+    Starts Jails
     """
-    if not jails and not rc:
-        ioc_common.logit({
-            "level"  : "ERROR",
-            "message": 'Usage: iocage start [OPTIONS] JAILS...\n'
-                       '\nError: Missing argument "jails".'
-        })
-        exit(1)
-
-    if rc:
-        ioc.IOCage(rc=rc, silent=True).start()
-    else:
-        for jail in jails:
-            ioc.IOCage(jail, rc=rc).start()
+
+    logger.print_level = log_level
+
+    for jail_identifier in jails:
+        jail = Jail.Jail(jail_identifier, logger=logger)
+        logger.log(f"Starting jail '{jail.humanreadable_name}'")
+        try:
+            jail.start()
+        except Exception as e:
+            exit(1)
```

Comment:
Sure it can by means of that error variable e (a good design would have it carry all relevant info)
If the library is logging it anyway then its cool